mythril/laser/ethereum/svm.py

```diff
@@ -151,17 +147,12 @@ def _execute_transactions(self, address):
         :return:
         """
         self.coverage = {}
-        for i in range(self.max_transaction_count):
-            initial_coverage = self._get_covered_instructions()
+        for i in range(self.transaction_count):
 
             self.time = datetime.now()
             logging.info("Starting message call transaction, iteration: {}".format(i))
             execute_message_call(self, address)
 
-            end_coverage = self._get_covered_instructions()
-            if end_coverage == initial_coverage:
```

Comment:
We can keep the initial_coverage and end_coverage so we can log.info the increased instruction coverage. WDYT?